yarn.lock

```diff
@@ -1655,32 +1655,6 @@
     rollup-pluginutils "^2.3.3"
     slash "^2.0.0"
 
-"@ovh-ux/manager-navbar@^0.5.0":
-  version "0.5.2"
-  resolved "https://nexus.ovh.net/nexus3/repository/npm-all/@ovh-ux/manager-navbar/-/manager-navbar-0.5.2.tgz#b8eb231b93482771cb1d1c442bd1ce0ed249ec1c"
-  integrity sha512-TcjFdhtB2yZiIWlqQNlkYpIWjQrJ5y00RBBgvfcpJbH7dwd70Ux5aVwrVGMKUkZcu7/3D5ucYU278lsbJkrC7w==
-  dependencies:
-    lodash "^4.17.11"
-    moment "^2.24.0"
-
-"@ovh-ux/manager-pci@^0.12.0":
```

Comment:
Diff results with a `yarn install` from the commit`92d58b5b1aa3f5c6ed8529adb07a4f8a7f8a062b`.